about.html

```diff
@@ -0,0 +1,133 @@
+            <p>The logo of ThunderWolf was created with different top designers.<br>
+            After online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize.<br>
+            After consultation with Mr. Agrawal we optimized the prize-winning-logo for this year's submit as following:</p>
+        </div>
+        <div>
+            <h2 class="detail-logo">TW</h2>
+        </div>
+    </div>
+    <div class="detail3">
+        <h2 class="detail3-title">See the past ThunderWolf Summits</h2>
+        <span class="guide-bar4"></span>
+        <p class="detail3-text">Take a look to the last two ThunderWolf Summits wich took place in Buenos Aires and Warsaw.</details></p>
+        <div class="detail3-images">
+            <img src="images/summit1.png" alt="summit1">
+            <img src="images/summit2.png" class="detail3-image2" alt="summit2">
+        </div>
+    </div>
+    <script src="script.js"></script>
+    <div class="site-banner1">
+        <div class="banner_black1">
+            <div>
+                <div class="ccl">
+                    <h2 class="partner1">Partner</h2>
+                    <span class="guide-bar"></span>
+                    <div class="banner-partners">
+                        <h3>mozilla</h3>
+                        <h3>Google</h3>
+                        <h3>NAVER</h3>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <footer class="site-footer1">
+        <div class="footer_black1">
+            <div>
+                <div class="ccl">
+                    <a rel="license" href="https://creativecommons.org/licenses/by/2.0/kr/">
+                        <img alt="license" class="license1" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" />
+                    </a>
+                    </div>
+                    <div class="footer-info">
+                        <p>2009 ThunderWold USA. Some Rights Reserved.<br /><br />
+                    </div>
+                </div>
+            </div>
+        </div>
+    </footer>
+    <footer class="site-footer">
+        <div class="footer_black2">
+            <div>
+                <div class="ccl">
+                    <a rel="license" href="https://creativecommons.org/licenses/by/2.0/kr/">
+                        <img alt="license" class="license3" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" />
+                    </a>
+                    </div>
+                    <div class="footer-info">
+                        <p>2009 ThunderWold USA. Some Rights Reserved.<br /><br />
+                        Rights are reserved for the ThunderWolf concerts.
+                        <br /><br />
+                        Not allow to share any information about logistic or any strategy of the concert.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] I suggest you give similar spacing, borders, shadows, and paddings to the original design.
![design](https://user-images.githubusercontent.com/44745855/176967944-5848d9d3-028c-4f73-bce4-f23f4483e57f.png)